builder-image/content_sets_rhel9.yml

```diff
@@ -0,0 +1,12 @@
+x86_64:
```

Comment:
nitpicking, all others YAML file use the `yaml` suffix, it's the only one afaict that ends with `.yml`

Reply:
This extension has been used since EAP 7.4. In the cekit doc it references .yml for content_set, I will stay on the safe side ;-).